applications/FluidDynamicsApplication/tests/TwoFluidStaticPoolTest/TwoFluidStaticPoolTest3D.json

```diff
@@ -0,0 +1,101 @@
+ {
+    "problem_data"                     : {
+        "problem_name"    : "TwoFluidStaticPoolTest3D",
+        "model_part_name" : "FluidModelPart",
+        "domain_size"     : 3,
+        "parallel_type"   : "OpenMP",
+        "echo_level"      : 0,
+        "start_time"      : 0.0,
+        "end_time"        : 0.1
+    },
+    "output_configuration"             : {
+        "result_file_configuration" : {
+            "gidpost_flags"       : {
+                "GiDPostMode"           : "GiD_PostBinary",
+                "WriteDeformedMeshFlag" : "WriteDeformed",
+                "WriteConditionsFlag"   : "WriteConditions",
+                "MultiFileFlag"         : "SingleFile"
+            },
+            "file_label"          : "time",
+            "output_control_type" : "time",
+            "output_frequency"    : 0.2,
+            "body_output"         : true,
+            "node_output"         : false,
+            "skin_output"         : false,
+            "plane_output"        : [],
+            "nodal_results"       : ["VELOCITY","PRESSURE","DISTANCE","DENSITY","DYNAMIC_VISCOSITY"],
+            "gauss_point_results" : []
+        },
+        "point_data_configuration"  : []
+    },
+    "restart_options"                  : {
+        "SaveRestart"      : "False",
+        "RestartFrequency" : 0,
+        "LoadRestart"      : "False",
+        "Restart_Step"     : 0
+    },
+    "solver_settings"                  : {
+        "solver_type"                 : "TwoFluids",
+        "model_import_settings"       : {
+            "input_type"     : "mdpa",
+            "input_filename" : "TwoFluidStaticPoolTest/TwoFluidStaticPoolTest3D"
+        },
+        "echo_level"                  : 1,
+        "compute_reactions"           : false,
+        "dynamic_tau"                 : 1.0,
+        "maximum_iterations"          : 10,
+        "relative_velocity_tolerance" : 1e-5,
+        "absolute_velocity_tolerance" : 1e-7,
+        "relative_pressure_tolerance" : 1e-5,
+        "absolute_pressure_tolerance" : 1e-7,
+        "volume_model_part_name"      : "Parts_Fluid",
+        "skin_parts"                  : [],
+        "no_skin_parts"               : [],
+        "time_stepping"               : {
+            "automatic_time_step" : false,
+            "time_step"           : 0.01
+        },
+        "linear_solver_settings" : {
+            "solver_type" : "SuperLUSolver",
+            "smoother_type" :"damped_jacobi",
+            "krylov_type" : "lgmres",
+            "coarsening_type" : "aggregation",
+            "max_iteration" : 500,
+            "provide_coordinates" : false,
+            "gmres_krylov_space_dimension" : 500,
+            "verbosity" : 0,
+            "tolerance" : 1e-8,
+            "scaling" : false,
+            "block_size" : 1,
+            "use_block_matrices_if_possible" : true,
+            "coarse_enough" : 500
+        }
+    },
+    "initial_conditions_process_list"  : [],
```

Comment:
this format is quite old already, please use the new format
this way this test won't have to be updated once the backwards-compatibility is removed

Reply:
Dear Philipp,

Reply:
Thank you very much for the hint. I hope that I could generate the correct file format. At least I do not get any more messages indicating depreciated entries when the case is configured...